### ISSUE TITLE
Search - Preview - Title - Unecessary data - Чайки 22.07.2017 

### ISSUE THREAD

**Issue by VasylynaF** [scholokov/long-travel-2#5511]
Steps to reproduce:
1. Open the page: https://test.long-travel.live/
2. Click on ''Travels'' in the header
3. Enter ''Чайки 22.07.2017'' in the search fiel

Actual result:
Trip name ''Чайки 22.07.2017
22.07.2017''

Expected result:
Trip name should be ''Чайки 22.07.2017''

Attachments:
![image](https://github.com/user-attachments/assets/be9aac09-18a1-472f-a301-12a02a312729)

Environment:
Windows 10 (x64)
Google Chrome 127.0.6533.120 (Офіційне складання) (64-розрядна версія)

**Comment by scholokov** [comment]
по прев'ю є чіткі вимоги, які елементи мають бути присутні

**Comment by VasylynaF** [comment]
@scholokov Так, але є ще подорожі Чайки інших років, де вказано в назві лише рік або дата під назвою, а тут дата якби дублюється 
![image](https://github.com/user-attachments/assets/564eb507-ff33-46d8-9ad8-48860b21090c)

**Comment by scholokov** [comment]
у нас є окремен поле "дата". яка має завжди бути присутня
і у нас є окремі вимоги до тайтлів (в деяких тайтлах є дата, для унікальності тайтла)

**Comment by VasylynaF** [comment]
@scholokov Я б у назві залишила лише рік, дату видно внизу, але якщо і так ок, тоді це не баг. Правильно?

**Comment by scholokov** [comment]
є унікальні подорожі, які відрізняються саме датою. бо там і рік і місяць тіж :)))))
тому логіка перевірки тайтлу - саме з докою по тайтлах.
хоча звісно що всюди можуть бути проблеми :)
